packages/eslint-plugin-pf-codemods/src/rules/helpers/getFromPackage.ts

```diff
@@ -54,19 +54,32 @@ function getSpecifiers<
   );
 }
 
-export function getFromPackage(context: Rule.RuleContext, packageName: string) {
+export function getFromPackage(
+  context: Rule.RuleContext,
+  packageName: string,
+  specifierNames?: string[]
+) {
   const astBody = context.getSourceCode().ast.body;
 
-  const imports = getSpecifiers<ImportDeclaration, ImportSpecifier>(
+  let imports = getSpecifiers<ImportDeclaration, ImportSpecifier>(
     astBody,
     "ImportDeclaration",
     packageName
   );
-  const exports = getSpecifiers<ExportNamedDeclaration, ExportSpecifier>(
+  let exports = getSpecifiers<ExportNamedDeclaration, ExportSpecifier>(
```

Comment:
It doesn't make a huge difference, but I think having these be non-mutated would keep things a _bit_ cleaner